### ISSUE TITLE
Should the tip 'Provide visible controls for audio and video players' be removed?

### ISSUE THREAD

**Issue by iadawn** [w3c/wai-quick-start#54]
The tip [Provide visible controls for audio and video players](http://w3c.github.io/wai-quick-start/designing.html#provide-visible-controls-for-audio-and-video-players) is not backed by specific WCAG requirements and there are no resources that we can point people to. An example is available but it doesn't do much more than what the text provides.

Is this an important enough tip to keep, despite it being different from the other tips?

**Comment by slhenry** [comment]
I do *not* think we should have this Tip. [medium-strong]
Rationale:
* What % of the audience for this page designs audio and video players? My guess is *very* low.
* If we don't point to resources for understanding what they need to do, it seems like a mean tease.

**Comment by melodyma** [comment]
I was actually looking for a resource on video player the other day. However, I would imagine most people just leverage existing video/audio players like youtube.

**Comment by hkramer** [comment]
My initial response was going to be an emphatic 'no' until I read the above. I think it is a sensible recommendation. What if "Provide visible controls ..." was changed to "Provide keyboard accessible controls ..." Then it could be linked to WCAG Rec: 2.1.1 All functionality of the content is operable through a keyboard interface..." and perhaps "Audio Control (1.4.2) although that might be a stretch.

**Comment by bakkenb** [comment]
I agree with Howard about SC 1.4.2 being a big stretch unless you qualified it with "plays automatically," which is not what I think you are going for here. That being said, I also agree with Howard that it could be tied to SC 2.1.1. To do that though I think it would be best to remove the word "visual" from the tip title.

**Comment by vmmiller** [comment]
I felt it was probably too much for the type of audience aimed at here.

**Comment by yatil** [comment]
Hm. When I was developing web pages custom video players were almost always part of the game. That said, the mockups usually did at least contain play pause buttons and volume controls, so I think the tip here seems rather obvious to designers.

I think that the keyboard navigation part could/should be mentioned in the tips for developing, in the [Ensure that all interactive elements are keyboard accessible](http://w3c.github.io/wai-quick-start/developing.html#ensure-that-all-interactive-elements-are-keyboard-accessible) tip.

**Comment by iadawn** [comment]
## Summary of feedback

* Remove
  * SLH+++, MM, HK, VM, EE
* Keep
  * [Crickets]
  
**Note**: Keyboard accessibility will be covered in Developing, Keyboard focus is covered in other Designing tip

**Comment by iadawn** [comment]
[EO Meeting on 3rd July 2015](http://www.w3.org/2015/07/03-eo-minutes.html#item03) resolved to remove this tip.
